### PR TITLE
Add data-migration-tool mirror

### PR DESCRIPTION
**Pull request by rhoerr**
Adds Terraform setup for https://github.com/mage-os/mirror-data-migration-tool based on the example of https://github.com/mage-os/terraform/pull/81

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -351,6 +357,12 @@ variable "repositories" {
       topics      = ["mage-os", "magento", "ecommerce", "magento2", "adobecommerce"]
     }
 
+    mageos-data-migration-tool = {
```
Comment (sprankhub) [inline variables.tf:360]: Do we need a mage-os-specific repository for this project? Do we plan to work on this? Do we release it?

Comment (rhoerr) [reply variables.tf:360]: @Vinai -- can the package be released to repo.mage-os.org without having our own version of it?

Comment (Vinai) [reply variables.tf:360]: Yes it can

**Comment by rhoerr** [comment]
Thanks, I removed the mageos- fork of it. PR sets up the mirror repository only now.

**Review by sprankhub — APPROVED** [review]

**Pull request merged**